tests/Makefile

```diff
@@ -195,11 +195,14 @@ $(TOP)/tools/mtouch/SdkVersions.cs: $(TOP)/tools/common/SdkVersions.cs.in
 	@$(MAKE) -C $(TOP)/src project-files
 	@touch $@
 
-xharness/xharness.exe: $(wildcard xharness/*.cs) xharness/xharness.csproj $(TOP)/tools/mtouch/SdkVersions.cs test.config test-system.config .stamp-src-project-files
+.stamp-build-mono-unit-tests: $(TOP)/.git/modules/external/mono/HEAD
```

Comment:
Just this part of my PR won't work, you also need the parts that call this target when needed:

https://github.com/xamarin/xamarin-macios/pull/5139/files#diff-00b1e42fbe85ec82bbf5d480ba69e88bR40
https://github.com/xamarin/xamarin-macios/pull/5139/files#diff-8f44b7ce671e35190b27341716ad7e8cR832
https://github.com/xamarin/xamarin-macios/pull/5139/files#diff-6f227b39f02c167ab381c32973b161ffR2408
https://github.com/xamarin/xamarin-macios/pull/5139/files#diff-2570283f10272437a1374f96e13c1cdbR21
etc.

Reply:
Solved in the latest push.